modules/libcom/src/error/errlog.c

```diff
@@ -638,49 +646,49 @@ static void errlogThread(void)
                 pvt.print = print = temp;
             }
 
-            pvt.nLost = 0u;
+            pvt.nLost = 0;
             epicsMutexUnlock(pvt.msgQueueLock);
 
             while(pos < print->pos) {
                 listenerNode *plistenerNode;
                 char* base = print->base + pos;
-                size_t mlen = epicsStrnLen(base+1u, pvt.bufSize - pos);
+                int mlen = (int)epicsStrnLen(base+1, pvt.bufSize - pos);
                 int stripped = 0;
 
                 if((base[0]&ERL_STATE_MASK) != ERL_STATE_READY || mlen>=pvt.bufSize - pos) {
-                    fprintf(stderr, "Logic Error: errlog buffer corruption. %02x, %zu\n",
+                    fprintf(stderr, "Logic Error: errlog buffer corruption. %02x, %u\n",
```

Comment:
> VxWorks does not support %z.

Could you make a make a minimal change to address just this?  eg. replace `%zu` with `%u` and cast the corresponding value to `(unsigned)`?

A small change like this could be easily reviewed and less contentious.  Potentially before the next Base release in about a month.  Mixing this up with functional changes is going to take longer.

Reply:
For the complete list of `%z` usages see https://bugs.launchpad.net/epics-base/+bug/1983472